src/Zend/Session.php

```diff
@@ -512,17 +512,7 @@ protected static function _checkId($id)
             }
         }
 
-        $hashBitsPerChar = ini_get('session.hash_bits_per_character');
-        if (!$hashBitsPerChar) {
-            $hashBitsPerChar = 5; // the default value
-        }
-        $pattern = '';
-        switch ($hashBitsPerChar) {
-            case 4: $pattern = '^[0-9a-f]*$'; break;
-            case 5: $pattern = '^[0-9a-v]*$'; break;
-            case 6: $pattern = '^[0-9a-zA-Z-,]*$'; break;
-        }
-        return preg_match('#' . $pattern . '#', $id);
+        return preg_match('#^[0-9a-zA-Z-,]*$#', $id);
```

Comment:
if `5` is the default value, shouldn't the regex here be `#^[0-9a-v]*$` to maintain previous behavior?